docs/recipes/ExpoRouter.md

```diff
@@ -1672,19 +1881,28 @@ We get that universal linking for free with `expo-router`!
 
 ## Code Cleanup
 
-Now that we have the boilerplate up and running again, let's clean some of the screen and navigation files that are no longer needed.
+Now that we have the boilerplate up and running again, let's clean some of the screen and navigation files that are no longer needed. We'll also remove the screen generator since there will not be one main directory for screens any longer
```

Comment:
```suggestion
Now that we have the boilerplate up and running again, let's clean some of the screen and navigation files that are no longer needed. We'll also remove the screen generator, since there will not be one main directory for screens any longer.
```